Aptoide/aptoide-client-v8
# [HOTFIX/MOB-201] Fix Darktheme RC

**What does this PR do?**

   This PR aims fixing the release candidate problems regarding the dark theme. These problems were reported from the design and QA team. 

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] RecommendedStoreWidget.java

**How should this be manually tested?**

Open the app and check all the problems reported in the ticket in both in light and dark theme.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-201](https://aptoide.atlassian.net/browse/MOB-201)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
